### ISSUE TITLE
Conflict with Caldera Connected Forms add on

### ISSUE THREAD

**Issue by KilakwaBT** [mecachisenros/caldera-forms-civicrm#113]
Working with the Caldera Connected forms add on. I create several forms and the processors with CiviCRM each works correctly. When I try to create the form which does the connections of the forms into a single form and try to save that form it just spins and does not save. However, if I disable CIviCRM and create the connector form and save it saves fine. If I then reactivate CiviCRM and use the new connected form it works as it should. Moving from form to form and processing correctly including CiviCRM processors.

To sum up I can get it to work by disabling CiviCRM when I need to create or modify the form which does the connections.

Using CiviCRM 5.13.2 with WordPress 5.2

**Comment by lyonsdigital** [comment]
Holy crap! I've been chasing this through my system for the past week. I can absolutely confirm this issue. I've been going crazy trying to find a ModSecurity note or an apache error. I was looking to post something else and just saw this issue. In the short term, I am building 3 separate forms where I could separate users with a connected form instead. Please fix this.

**Comment by mecachisenros** [comment]
@KilakwaBT @lyonsdigital thanks for reporting, CFC is not currently compatible with Connected Forms. I don't know what time it will take to fix it/integrate it so I don't know when it will be fixed, but feel free to take a stab at it, PRs are always welcome, equally ping me here if you're willing to sponsor this issue.
